dayrepo/snippets/urls.py

```diff
@@ -14,5 +14,6 @@
     path("checklist/", views.checklist_post,name="checklist_post"),
     path("edit/<int:snippet_id>/", views.snippet_edit,name="snippet_edit"),
     path("edit/checklist/<int:checklist_id>/", views.checklist_edit,name="checklist_edit"),
+    path("delete/<int:post_id>/<str:post_type>/", views.post_delete,name="post_delete"),
```

Comment:
```suggestion
    path("delete/<int:target_id>/<str:delete_type>/", views.post_delete,name="post_delete"),
```

他のコードの箇所では post_id と checklist_id は明確に別物として扱っていて、その上でこのパスにはどっちが渡ってくるかわからなくなっているので、1段階抽象度を上げた命名にするといいと思う〜

あと post_type だと postの種類って意味になるけど、あくまでも削除する種類だと思うのでこっちは delete ってつけた方がわかりやすいと思う〜

Reply:
viewの記載も変える必要があるためこちらで対処しました。
https://github.com/bob-g12/dayrepo-for-transportation/pull/56/commits/774bd07638c22c167e4a9aab8cd276b93c33afc1